### DISCUSSION TITLE
Some ICON's

### DISCUSSION THREAD

**Discussion by AviadorLP** [wills106/homeassistant-solax-modbus#29] [Ideas]
I always change these icons for start and end times.
START mdi:clock-start
END  mdi:clock-end

I like icon diversity so its not always the same
maybe you think its a good ideia, if not sorry to bother you.

Great work on this integration
Always looking for your updates

**Comment by wills106** [comment]
Do you mean for the following:

select.solax_charger_start_time_1
select.solax_charger_end_time_1

etc

It's still work in progress, but I am open to ideas and improvements :)

**Reply by AviadorLP** [reply]
actually i meant in:
sensor.solax_charger_start_time_1
sensor.solax_charger_end_time_1
sensor.solax_charger_start_time_2
sensor.solax_charger_end_time_2 
sensor.solax_backup_charge_start
sensor.solax_backup_charge_end

but if you think it will look cool in the select options its your choice ;)

i also use in sensor.solax_rtc mdi:clock-digital
i use this sensor to fast check when all the sensors where last updated.
they usually all update at the same time more or less.

**Comment by cleveHEX** [comment]
I see someone had this idea before me, you can checkout #611 and suggest anything.
